spec/Features/users_spec.rb

```diff
@@ -0,0 +1,9 @@
+require "spec_helper"
+
+feature "User sign up" do
+	scenario "I can sign up as a new user" do
+		visit "/"
+		expect { sign_up }.to change(User, :count).by(1)
+		expect(User.first.email).to eq("eleni_s_2015@hotmail.com")
+end
```

Comment:
Lint/BlockAlignment: `end` at 8, 0 is not aligned with `scenario "I can sign up as a new user" do` at 4, 1.